aptihramy-backend/database/database.py

```diff
@@ -220,3 +234,21 @@ def get_tracked_features(self) -> tuple[list[str], list[str]]:
         raw = [feature.name for feature in self._record_schema.fields]
         pretty = [COLUMN_RAW_TO_PRETTY[col] for col in raw]
         return (raw, pretty)
+
+    def get_record(self, frame_idx: int, record_idx: int) -> pl.DataFrame:
```

Comment:
It is clearer to return a list (ex: `list[bb.Element]`) or define a type `Record = list[bb.Element]` rather than returning a one row dataframe (as then the type doesn't indicate that it only contains one row that represent a record).